Makefile

```diff
@@ -56,10 +59,11 @@ links:
 checks: tidy ws links
 
 test:
-	${GOTESTSUM_CMD} ${PACKAGES} -- -timeout=${LOCAL_TIMEOUT} -short
+	${GOTESTSUM_CMD} --packages "${TEST_PACKAGES}" -- -timeout=${LOCAL_TIMEOUT} ${SHORT_FLAG}
+	${GOTESTSUM_CMD} --packages ./acceptance/... -- -timeout=${LOCAL_TIMEOUT} ${SHORT_FLAG} -run ${ACCEPTANCE_TEST_FILTER}
 
 test-slow:
-	${GOTESTSUM_CMD} ${PACKAGES} -- -timeout=${LOCAL_TIMEOUT}
+	make test SHORT_FLAG="-short"
```

Comment:
Reuse the main "test" target.

Reply:
Should be the opposite? SHORT_FLAG="" for test-slow and SHORT_FLAG="-short" for test.

The "reuse" by calling make recursively IMO is not worth it I'd just copy paste the two lines directly with appropriate flags. Maybe I'm running /usr/local/bin/make on purpose?